src/envs/cover.py

```diff
@@ -382,10 +382,10 @@ def get_name(cls) -> str:
     def options(self) -> Set[ParameterizedOption]:
         return {self._Pick, self._Place}
 
-    @staticmethod
-    def _Pick_policy(s: State, m: Dict, o: Sequence[Object],
+    def _Pick_policy(self, s: State, m: Dict, o: Sequence[Object],
                      p: Array) -> Action:
         del m  # unused
+        _ = self  # unused
```

Comment:
can't use del here, pylint complains that method could be staticmethod bc self is unused